php-checks/src/main/java/org/sonar/php/checks/UnusedFunctionParametersCheck.java

```diff
@@ -82,7 +83,7 @@ private void checkParameters(FunctionTree tree) {
    */
   public boolean isExcluded(MethodDeclarationTree tree) {
     MethodSymbol methodSymbol = Symbols.get(tree);
-    return !tree.body().is(Tree.Kind.BLOCK) || !(methodSymbol.isOverriding().isFalse());
+    return !tree.body().is(Tree.Kind.BLOCK) || !(methodSymbol.isOverriding().isFalse()) || methodSymbol.owner().is(ClassSymbol.Kind.ABSTRACT);
```

Comment:
What about private methods in abstract classes? SONARPHP-1031 only mentions public methods.

Reply:
I think I would also add protected methods to the excluded methods